### ISSUE TITLE
Accessibility issues with Talkback

### ISSUE THREAD

**Issue by SaschaCowley** [andOTP/andOTP#498]
#### General information

* **App version:** 0.7.0-Play
* **App source:** Google Play
* **Android Version:** 10
* **Custom ROM:** None (Google Pixel 4)

#### Issues

* When focusing the FAB with TalkBack, it should be labelled as "add..." or suchlike, whereas it it is currently reported as being unlabelled.
* When the FAB is pressed and the overlay shown:
    - One of the options displayed should be focused;
    - The user should not be able to swipe to any of the user interface elements that are hidden behind the overlay;
    - Focusing the FAB or overlay should indicate that activating will close the overlay, whereas currently the FAB is unlabelled and the overlay reports nothing.
* The buttons in token cards should be labelled, whereas currently they are reported as unlabelled.
    - At a minimum, they should be called something like "copy token" and "more options", respectively.
    - Optimally, they should be labelled something like "Copy \<service-name\> token" and "More options for \<service-name\>".

#### Steps to reproduce
 
 * Activate TalkBack (in accessibility settings) and attempt to use the andOTP app.

**Comment by SaschaCowley** [comment]
The icons in the "change image" dialog are also unlabelled. While it would be nice to have them labelled (and should easily be possible, I think), this should not be given as high a priority as the issues listed in my initial report. It is important to remember that not only totally blind users use screen readers, though, so having everything, even what what one might think of as purely visual features, working as well as possible with screen readers, is very important.

**Comment by flocke** [comment]
I fixed what I could myself (see your original comments) but most of the things related to the FAB menu can only be fixed by the upstream project that is used. I opened an issue there: https://github.com/leinardi/FloatingActionButtonSpeedDial/issues/149. Please go there for any further discussions regarding the FAB menu.

**Comment by ziegenberg** [comment]
There is now a PR for solving the issue upstream: https://github.com/leinardi/FloatingActionButtonSpeedDial/pull/151

**Comment by flocke** [comment]
Yeah, that but I haven't found the time to test this yet. Maybe you could find some time to test it and report back to upstream?

**Comment by flocke** [comment]
I finally remembered to test the changes and they are working nicely for the most part. There are still two points that may need fixing but after that I believe it should be fine.
I hope we manage to get it in for the next release, but we will have to see (that's why I cautiously moved the issue to the next milestone).

**Comment by ziegenberg** [comment]
PR  #843 updates FloatingActionButtonSpeedDial to version 3.2.0 which fixes most of the remaining issues.

I tested when the FAB is pressed and the overlay is shown:

- [x] One of the options displayed should be focused (it's the closest option to the FAB)
- [ ] The user should not be able to swipe to any of the user interface elements that are hidden behind the overlay;
- [x] Focusing the FAB or overlay should indicate that activating will close the overlay, whereas currently the FAB is unlabelled and the overlay reports nothing.

**Comment by ziegenberg** [comment]
With version 3.2.0 the icons in the ActionItems disappear. 😢 Guess we found a new bug.
